src/sentry/migrations/0001_initial.py

```diff
@@ -864,7 +864,6 @@ class Migration(migrations.Migration):
                 'verbose_name': 'message',
                 'verbose_name_plural': 'messages',
             },
-            bases=(sentry.models.event.EventCommon, models.Model),
```

Comment:
what does this actually do ?

Reply:
These list the classes the model inherits from. I... assume that it didn't really do anything since inheriting from EventCommon had no bearing on the model creation (and it will inherit from models.Model by default).